src/code/padmgr.c

```diff
@@ -4,55 +4,55 @@
 s32 D_8012D280 = 1;
 
 OSMesgQueue* PadMgr_LockSerialMesgQueue(PadMgr* padMgr) {
-    OSMesgQueue* ctrlrQ = NULL;
+    OSMesgQueue* ctrlrQueue = NULL;
 
     if (D_8012D280 > 2) {
         // "serialMsgQ Waiting for lock"
         osSyncPrintf("%2d %d serialMsgQロック待ち         %08x %08x          %08x\n", osGetThreadId(NULL),
-                     padMgr->serialMsgQ.validCount, padMgr, &padMgr->serialMsgQ, &ctrlrQ);
+                     MQ_GET_COUNT(&padMgr->serialEventQueue), padMgr, &padMgr->serialEventQueue, &ctrlrQueue);
     }
 
-    osRecvMesg(&padMgr->serialMsgQ, (OSMesg)&ctrlrQ, OS_MESG_BLOCK);
+    osRecvMesg(&padMgr->serialEventQueue, (OSMesg*)&ctrlrQueue, OS_MESG_BLOCK);
 
     if (D_8012D280 > 2) {
         // "serialMsgQ Locked"
         osSyncPrintf("%2d %d serialMsgQをロックしました                     %08x\n", osGetThreadId(NULL),
-                     padMgr->serialMsgQ.validCount, ctrlrQ);
+                     MQ_GET_COUNT(&padMgr->serialEventQueue), ctrlrQueue);
     }
 
-    return ctrlrQ;
+    return ctrlrQueue;
 }
 
-void PadMgr_UnlockSerialMesgQueue(PadMgr* padMgr, OSMesgQueue* ctrlrQ) {
+void PadMgr_UnlockSerialMesgQueue(PadMgr* padMgr, OSMesgQueue* ctrlrQueue) {
     if (D_8012D280 > 2) {
         // "serialMsgQ Unlock"
         osSyncPrintf("%2d %d serialMsgQロック解除します   %08x %08x %08x\n", osGetThreadId(NULL),
-                     padMgr->serialMsgQ.validCount, padMgr, &padMgr->serialMsgQ, ctrlrQ);
+                     MQ_GET_COUNT(&padMgr->serialEventQueue), padMgr, &padMgr->serialEventQueue, ctrlrQueue);
     }
 
-    osSendMesg(&padMgr->serialMsgQ, ctrlrQ, OS_MESG_BLOCK);
+    osSendMesg(&padMgr->serialEventQueue, (OSMesg)ctrlrQueue, OS_MESG_BLOCK);
```

Comment:
Here `ctrlrQueue` is the `sSerialEventQueue` as created in Main() and passed to `PadMgr_Init` as `serialEventQueue`
`padMgr->serialEventQueue` only holds `serialEventQueue`, it isn't `serialEventQueue`

I'm not very inspired for names...
```suggestion
    osSendMesg(&padMgr->serialEventQueueLockQueue, (OSMesg)serialEventQueue, OS_MESG_BLOCK);
```

Reply:
I've attempted to make this less ambiguous for now, we can probably revisit in a more padmgr-focused setting